lnwallet/chanfunding/coin_select_test.go

```diff
@@ -73,18 +74,18 @@ func TestCoinSelect(t *testing.T) {
 				{
 					TxOut: wire.TxOut{
 						PkScript: p2wkhScript,
-						Value:    1 * btcutil.SatoshiPerBitcoin,
+						Value:    1 * coin,
```

Comment:
changes like this should be squashed with the original commit 😄

Reply:
I have made the cosmetic change to squash this with its corresponding commit ;)